src/devices/machine/8530scc.cpp

```diff
@@ -638,4 +643,60 @@ SCC: port B reg 15 write 0x08     DCD interrupt enable
 SCC: port B reg 1 read 0x09   Rx IRQ on first char or special, ext int enable
 SCC: port B reg 3 write 0xdd      Rx 8 bits, enter hunt mode, CRC enable, address search mode, Rx enable
 
+x68000: port A rs232, port B mouse
```

Comment:
I don’t think adding logs for modes that are implemented is all that useful.  The AppleTalk check log serves as a reminder that SDLC mode isn’t implemented.

Reply:
Done.